conda/prod.yaml

```diff
@@ -18,6 +18,8 @@ dependencies:
   - psycopg2
   - lightgbm
   - loguru
+  - pangres
+  - sodapy
```

Comment:
sodapy is not available on conda-forge yet. for now we can add it to the pip section.
PS: I am doing it now